### PR TITLE
Add SailfishOS

### PR DESCRIPTION
**Pull request by olpeh**
https://github.com/sailfishos-community/awesome-sailfishos

A curated list of awesome Sailfish OS resources.

Sailfish OS is a mobile operating system based on open source and developed by the Finnish mobile company Jolla Ltd. and the Sailfish OS community.

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -102,6 +102,7 @@
 - [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion) - Write cross-platform native apps for iOS, Android, macOS, tvOS, and watchOS in Ruby.
 - [Smart TV](https://github.com/vitalets/awesome-smart-tv) - Create apps for different TV platforms.
 - [GNOME](https://github.com/Kazhnuz/awesome-gnome) - Simple and distraction-free desktop environment for Linux.
+- [Sailfish OS](https://github.com/sailfishos-community/awesome-sailfishos) - True Independent Mobile OS
```
Comment (aschrijver) [inline readme.md]: I think your description could be improved. This sounds like a marketing statement (and should start with 'Truly', not 'True'). Maybe: "A truly independent open-source mobile operating system."

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

File: readme.md [hunk 2]
```diff
@@ -102,6 +102,7 @@
 - [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion) - Write cross-platform native apps for iOS, Android, macOS, tvOS, and watchOS in Ruby.
 - [Smart TV](https://github.com/vitalets/awesome-smart-tv) - Create apps for different TV platforms.
 - [GNOME](https://github.com/Kazhnuz/awesome-gnome) - Simple and distraction-free desktop environment for Linux.
+- [Sailfish OS](https://github.com/sailfishos-community/awesome-sailfishos) - Truly Independent Mobile OS
```
Comment (sindresorhus) [inline readme.md]: The description should end in a `.`

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/sailfishos-community/awesome-sailfishos
✖ Linting

  README.md:36:49
  ✖   19:1   ToC item "Tips and Tricks" does not match corresponding heading "Deploy"  remark-lint:awesome/toc
  ✖   36:49  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   37:61  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   39:62  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   41:63  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   43:68  List item link and description separated by invalid whitespace            remark-lint:awesome/list-item
  ✖   45:82  List item link and description separated by invalid whitespace            remark-lint:awesome/list-item
  ✖   50:60  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   52:53  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   54:48  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   56:51  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   59:50  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   63:80  List item link and description separated by invalid whitespace            remark-lint:awesome/list-item
  ✖   65:77  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖   66:81  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖  114:54  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖  115:49  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖  116:58  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖  120:46  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖  121:83  List item description must end with proper punctuation                    remark-lint:awesome/list-item
  ✖  138:79  List item link and description separated by invalid whitespace            remark-lint:awesome/list-item

  21 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**